docs/topics/resources/spring-boot/reactive-examples/example-http-service-webflux-pom.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>dev.snowdrop</groupId>
+    <artifactId>vertx-spring-boot-samples</artifactId>
+    <version>0.0.3-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>vertx-spring-boot-sample-http</artifactId>
+
+  <dependencies>
+      <!-- tag::http-service-pom-dependencies[] -->
+    <dependency>
+      <groupId>dev.snowdrop</groupId>
+      <artifactId>vertx-spring-boot-starter-http</artifactId>
+      <version>${project.version}</version>
```

Comment:
I know this was taken from the sample. But because this documentation assumes that `spring-boot-bom` is used, `version` tag is not necessary here.
If this whole pom.xml is exposed somewhere, we should rewrite it to look the way it should from the documentation perspective.

Reply:
The full POM files for examples are maintained in the docs repo, but are currently not exposed.
I'll rewrite the examples to remove the version property references, because the documented procedure also assumes that `spring-boot-bom` is imported in the POM file.

Reply:
removed version references form POM examples